Sources/Interaction/Widgets/OrientationMarkerWidget/index.js

```diff
@@ -81,20 +86,25 @@ function vtkOrientationMarkerWidget(publicAPI, model) {
       selfRenderer.addViewProp(model.actor);
       model.actor.setVisibility(true);
 
-      selfRenderer.setViewport(...getViewport());
-
       const { unsubscribe } = model.interactor.onAnimation(
-        updateMarkerOrientation
+        publicAPI.updateMarkerOrientation
       );
       interactorUnsubscribe = unsubscribe;
 
+      window.addEventListener('resize', publicAPI.updateViewport);
+
+      publicAPI.updateViewport();
+      publicAPI.updateMarkerOrientation();
+
       model.enabled = true;
     } else {
       if (!model.enabled) {
         return;
       }
       model.enabled = false;
 
+      window.removeEventListener('resize', publicAPI.updateViewport);
```

Comment:
I don't think we want to bind listener here...

Reply:
wait... I see... It might be fine, but better we may be able to listen to vtkXXXObject modified or if we have a size change event. That way, we don't rely on the window...

If the user update the size by hand for example...